### PR TITLE
Network set

### PR DESCRIPTION
**Pull request by supreethpy**
### Description
Adding support for Network set to be able to pull their status for example for monitoring purposes

### Issues Resolved
None

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -120,13 +120,13 @@
...
+|<sub>/rest/network-sets</sub>                                                            | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   | :white_check_mark:   |
+|<sub>/rest/network-sets</sub>                                                            | POST     | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   | :white_check_mark:   |
+|<sub>/rest/network-sets/withoutEthernet</sub>                                            | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_multiplication_x:   |:heavy_multiplication_x:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   |:heavy_multiplication_x:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | PUT      | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   |:heavy_multiplication_x:   |
```
Comment (AnushaY1916) [inline endpoints-support.md]: Why is PUT, DELETE and GET by ID is mentioned as not supported ?

Comment (jyothis-gireesan) [reply endpoints-support.md]: same

Comment (supreethpy) [reply endpoints-support.md]: Done

**Comment by supreethpy** [comment]
Merging the Network_Set resource to main branch

**Pull request merged**